rabbitmq/tpl/rabbitmq-rc.yaml

```diff
@@ -0,0 +1,66 @@
+#helm:generate helmc tpl -d tpl/values.toml -o manifests/rabbitmq-rc.yaml $HELM_GENERATE_FILE
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: rabbitmq
+  labels:
+    app: rabbitmq
+    provider: rabbitmq-server
+    heritage: bitnami
+spec:
+  replicas: 1
+  selector:
+    app: rabbitmq
+    provider: rabbitmq-server
+    version: 3.6.2-r0
+  template:
+    metadata:
+      labels:
+        app: rabbitmq
+        provider: rabbitmq-server
+        version: 3.6.2-r0
+        heritage: bitnami
+    spec:
+      containers:
+      - name: rabbitmq
+        image: bitnami/rabbitmq:3.6.2-r0
+        env:
+        - name: RABBITMQ_USERNAME
+          value: {{ .rabbitmqUsername | quote }}
+        - name: RABBITMQ_PASSWORD
+          value: {{ .rabbitmqPassword | quote }}
+        - name: RABBITMQ_ERLANGCOOKIE
+          value: {{ .rabbitmqErlangcookie | quote }}
+        - name: RABBITMQ_NODEPORT
+          value: {{ .rabbitmqNodeport | quote }}
+        - name: RABBITMQ_NODETYPE
+          value: {{ .rabbitmqNodetype | quote }}
+        - name: RABBITMQ_NODENAME
+          value: {{ .rabbitmqNodename | quote }}
+        - name: RABBITMQ_CLUSTERNODENAME
+          value: {{ .rabbitmqClusternodename | quote }}
+        - name: RABBITMQ_VHOST
+          value: {{ .rabbitmqVhost | quote }}
+        - name: RABBITMQ_MANAGERBINDIP
+          value: {{ .rabbitmqManagerbindip | quote }}
+        ports:
+        - name: rabbitmq
+          containerPort: 15672
+        livenessProbe:
+          httpGet:
```

Comment:
@Fortiz2305 I wonder if `rabbitmqctl status` is a better check for the liveness and readiness probes. I'm not a rabbitmq expert, so I may be wrong here